src/Tribe/Main.php

```diff
@@ -3839,7 +3839,9 @@ public function isVenue( $postId = null ) {
 		public function isOrganizer( $postId = null ) {
 			if ( $postId === null || ! is_numeric( $postId ) ) {
 				global $post;
-				$postId = $post->ID;
+				if ( isset( $post->ID ) ) {
```

Comment:
We could also change the whole thing to just 
```
$postId = get_post_id();
```
Which allows us to remove the global var here and simplify the code.